src/haz3lcore/zipper/action/Action.re

```diff
@@ -8,10 +8,15 @@ type move =
   | Local(planar)
   | Goal(Measured.Point.t);
 
+[@deriving (show({with_path: false}), sexp, yojson)]
+type jump_target =
+  | TileId(Id.t)
+  | BindingSiteOfIndicatedVar;
+
 [@deriving (show({with_path: false}), sexp, yojson)]
 type t =
   | Move(move)
-  | JumpToId(Id.t)
+  | Jump(jump_target)
```

Comment:
Now that I've been doing some similar in Select, I'm somewhat tempted to consolidate these actions under Move.... make these Move(Id(id)), Move(ToDefinition), and make existing Move(whatever) to Move(Relative(whatever))... not quite sure how I feel about this change though. @cyrus- thoughts?